eng/tools/dependency-testing/index.js

```diff
@@ -281,8 +292,6 @@ async function updateRushConfig(repoRoot, targetPackage, testFolder) {
   rushSpec.projectFolderMaxDepth = 5;
   const rushPath = path.resolve(path.join(repoRoot, "rush.json"));
   await packageUtils.writePackageJson(rushPath, rushSpec);
-  console.log("rush spec -");
```

Comment:
Why is this needed?

Reply:
You mean removing the cnsole.log for rush.json? I wanted to remove it from the logs to avoid the noise being added. Initially I had it in there so that I could verify if anything goes wrong just by looking at the logs. Now over 1+ years I have the assurance that this is working right.

Reply:
Ok that makes sense! Thanks!